webserver/src/webserver.cpp

```diff
@@ -1,3 +1,7 @@
+// Ensure that Mongoose API is defined the same way in all translation units!
+#include <webserver/mongoose.h> // IWYU pragma: keep
```

Comment:
This was weird to debug. Basically, if C++ headers are included before mongoose, there'll be some weird issues and the `static_assert` in `mongooseserver.h` will fire... Which shouldn't because the typedef and all that remains the exact same thing. Digging through a Clang AST I was able to find out that somewhere deep `__falign_t` becomes defined in a different way.

Without the static assert, the code compiles, and I did not find any issues run-time, but it's better to be ODR-safe.